### ISSUE TITLE
/opt/solr issue when migrating Circle CI from Precise-->Trusty

### ISSUE THREAD

**Issue by cclauss** [ckan/ckan#4347]
Fixes #4335 
Unblocks #4060, #4345 

### CKAN Version if known (or site URL)
https://github.com/ckan/ckan

### Please describe the expected behavior
Can successfully migrate Circle CI builds from __Ubuntu 12.04 (Precise)__ --> __Ubuntu 14.04 (Trusty)__

### Please describe the actual behavior
CircleCI failing on step "__cp -R /opt/solr-4.3.1 $SOLR_HOME__" #4335 because /opt is empty.

### What steps can be taken to reproduce the issue?
1. Create a new branch
2. Go to https://circleci.com/gh/__YourGitHubID__/ckan/edit#build-environment
3. Click on __Ubuntu 14.04 (Trusty)__
4. Commit __any change__ to that branch to force Circle CI to build

**Comment by cclauss** [comment]
Closing in favor of #4394
